src/CoreBundle/private/library/classes/TCMSLogChange.class.php

```diff
@@ -913,6 +915,16 @@ public static function GetMessageTypeByName($sMessageTypeName)
         return $sRet;
     }
 
+    public static function getMessageTypeIdBySystemName(string $messageTypeName): ?string
```

Comment:
This method c/should document which system names there are normally.

Reply:
I've added a comment with the default available system names.